src/support/mod.rs

```diff
@@ -123,6 +124,15 @@ pub unsafe fn shutdown_llvm() {
     LLVMShutdown()
 }
 
+pub fn parse_command_line_options(argc: i32, argv: &[&str], overview: &str) {
```

Comment:
What does this do? Could you add some docs? Is it safe to call more than once?

Reply:
Actually we haven't tested in yet as we have some blockers. I'll remove this function from this patch and later create another one.